ckanext/datavic_iar_theme/grunt/sass/_header.scss

```diff
@@ -252,3 +252,9 @@ $rpl-site-header-logout-btn-icon-margin: 0 0 0 $rpl-space-2 !default;
     }
   }
 }
+
+.module .dashboard-header {
+  padding: 15px;
+  font-size: 1.5em;
+  padding: 0px 15px 0px 15px;
```

Comment:
1. You should not define the padding twice in the same rule.
2. Heading is not aligned with the page content